README.md

```diff
@@ -37,9 +41,27 @@ setenv FILEMGR_HOME $OODT_HOME/filemgr
 setenv PGE_ROOT $OODT_HOME/pge
 setenv PCS_HOME $OODT_HOME/pcs
 ```
-
-Installation 
-============ 
+Automated Install
+=================
+1. Navigate to desired location for imagecat
+2. git clone https://github.com/chrismattmann/imagecat.git
+3. cd imagecat
+4. cd auto
```

Comment:
Can you update the folder in your pull request to be lowercase auto instead of "Auto" (note folder path)